iothub_client/src/iothub_client_authorization.c

```diff
@@ -20,6 +20,7 @@
 
 #define DEFAULT_SAS_TOKEN_EXPIRY_TIME_SECS          3600
 #define INDEFINITE_TIME                             ((time_t)(-1))
+#define MIN_SAS_EXPIRY_TIME                         30  // 30 seconds
```

Comment:
>MIN_SAS_EXPIRY_TIME [](start = 8, length = 19)

Why not 5 seconds? Or 10 seconds? 
Do we document the default of this SAS token refresh someplace in our header comments or devdocs?

Reply:
As soon as I typed this, our e2e tests failed because they set minimum 10 seconds for fault injection :) Please set this to 5 seconds. It should be fine. 

---
In reply to: [255260962](https://github.com/Azure/azure-iot-sdk-c/pull/857#discussion_r255260962) [](ancestors = 255260962)

Reply:
30 seconds is arbitrary.  I had to pick a number.  I changed it to 5 seconds.

---
In reply to: [255263674](https://github.com/Azure/azure-iot-sdk-c/pull/857#discussion_r255263674) [](ancestors = 255263674,255260962)